### DISCUSSION TITLE
When version 2 release date?

### DISCUSSION THREAD

**Discussion by trippo** [Power-Components/livewire-powergrid#204] [Q&A] [answered]
I'm starting a new project with alpine 3 and tailwind 3, I want to know when version 2 will be available.

Currently, can I use dev branch to start developing?

**Comment by luanfreitasdev** [comment, marked as answer]
Hi @trippo . The dev branch is almost ready and scheduled to be released next week.

I'm talking about some tests and I'd like to hear your opinion if you install it first.

I also have to update the documentation, mainly on updating from v1 and on the new totalRow functionality (withSum, count, avg) that I need to revise.

https://livewire-powergrid.docsforge.com/dev/upgrade-from-v1/

https://livewire-powergrid.docsforge.com/dev/cell-action-buttons/#withsumstring-label-sum-bool-header-false-bool-footer-false

powergrid-demo I've already done a pull request with tailwind v3 and it worked great

https://github.com/Power-Components/powergrid-demo/pull/5

Thanks.
